SpatialGDK/Source/SpatialGDKTests/SpatialGDK/EngineClasses/GridBasedLBStrategy/GridBasedLBStrategyTest.cpp

```diff
@@ -0,0 +1,283 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "CoreMinimal.h"
+#include "Engine/World.h"
+#include "EngineClasses/GridBasedLBStrategy.h"
+#include "GameFramework/DefaultPawn.h"
+#include "GameFramework/GameStateBase.h"
+#include "SpatialConstants.h"
+#include "TestDefinitions.h"
+#include "TestGridBasedLBStrategy.h"
+#include "Tests/AutomationCommon.h"
+#include "Tests/AutomationEditorCommon.h"
+
+#define GRIDBASEDLBSTRATEGY_TEST(TestName) \
+	TEST(Core, UGridBasedLBStrategy, TestName)
+
+UWorld* TestWorld;
```

Comment:
Can we put these in an anon namespace.
(or make them static but I think we've preferred anon namespaces elsewhere).

Reply:
Done